cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -106,35 +146,28 @@ func (m *Manager) preview(ctx context.Context, interactive bool) (*PreviewResult
 		previewResult = result
 
 		return nil
-	}
-
-	spinner := spin.NewSpinner("Preparing infrastructure provisioning")
-	defer spinner.Stop()
-
-	err := previewAndReportProgress(spinner)
+	})
 
 	if err != nil {
 		return nil, fmt.Errorf("previewing infrastructure: %w", err)
 	}
 
-	spinner.Println("Prepared infrastructure provisioning")
+	m.console.Message(ctx, "Prepared infrastructure provisioning")
```

Comment:
Should not show anything if interactive == false....

Reply:
Great feedback @karolz-ms.  I need to do some cleanup on the interactive stuff.

Reply:
Updated so spinner instances are only ever created and used when in interactive mode.